src/main/java/com/minecolonies/coremod/items/ItemChorusBread.java

```diff
@@ -0,0 +1,76 @@
+package com.minecolonies.coremod.items;
+
+import com.ldtteam.structurize.util.LanguageHandler;
+import com.minecolonies.api.creativetab.ModCreativeTabs;
+import com.minecolonies.api.util.constant.TranslationConstants;
+import com.minecolonies.coremod.util.TeleportHelper;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.Food;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.Style;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraft.world.World;
+import net.minecraft.world.dimension.DimensionType;
+import static com.minecolonies.api.util.constant.Constants.STACKSIZE;
+import java.util.List;
+
+/**
+ * Class handling Chorus Bread.
```

Comment:
Not sure if I'd actually call this javadoc :D can you describe a little what the item is for here instead? same with other item classes

Reply:
Fixed